src/parse/formats.js

```diff
@@ -70,12 +70,11 @@ export const html = (opt) => {
   return pumpify.obj(xml2json({ ...opt, strict: false }), json(opt))
 }
 
-export const shp = () => {
-  const head = through2()
-  const mid = shpToJSON(head)
-  const tail = JSONStream.parse('features.*')
-  return duplexify.obj(head, pump(mid, tail))
-}
+export const gdb = () => from('gdb')
```

Comment:
These will all take options, so should be more like `const gdb = from.bind(null, 'gdb')`